yaml/com/ibm/VPD/FRUManager.interface.yaml

```diff
@@ -0,0 +1,122 @@
+description: >
+    Implement to manage VPD in the system.
+
+methods:
+    - name: WriteKeyword
+      description: >
+          Method to update the keyword value with the given value. Updates the
```

Comment:
The description is still not correct.
It should explain how it behaves in case of different paths given.

Reply:
updated